showcase/app/templates/layouts/app-frame/index.hbs

```diff
@@ -182,7 +182,7 @@
 
   <Shw::Text::H2>Content</Shw::Text::H2>
 
-  <Shw::Grid @columns={{1}} {{style gap="2rem"}} as |SG|>
+  {{!-- <Shw::Grid @columns={{1}} {{style gap="2rem"}} as |SG|>
```

Comment:
I was thinking that these non-framed examples may not be useful/needed so I tried commenting them out and leaving just the framed examples.

Let me know what you think and I'll either go ahead and delete these or else put them back.

Reply:
I see your point. Given that we can't do visual regression testing on them, but we may in the future (I just have to understand how to solve a problem in how the URLs are computed in the tests) if it's OK for you I would prefer to leave the code commented out (it's easy to remove it if we want, one day).

Reply:
Ok thanks, so I will leave as is with the code commented out. I will just add a comment with an explanation.